lbry/extras/daemon/daemon.py

```diff
@@ -3142,8 +3142,8 @@ def jsonrpc_channel_list(self, account_id=None, wallet_id=None, page=None, page_
                     f"Use --allow-duplicate-name flag to override."
                 )
 
-        file_path = await self._video_file_analyzer.verify_or_repair(validate_file, optimize_file, file_path)
-
+        file_path = await self._video_file_analyzer.verify_or_repair(validate_file, optimize_file,
+                                                                     file_path, ignore_non_video=True)
```

Comment:
We try to avoid the "pan handle" style. Can you move the arguments to be on a new line:

```
        file_path = await self._video_file_analyzer.verify_or_repair(
            validate_file, optimize_file, file_path, ignore_non_video=True
        )
```

Reply:
Fixed